wdl/MELT.wdl

```diff
@@ -600,12 +600,16 @@ task RunMELT {
     set -Eeuo pipefail
 
     # MELT expects the BAM index to have extension ".bam.bai"
-    mv ~{bam_file} ~{sample_id}.bam
-    mv ~{bam_index_file} ~{sample_id}.bam.bai
+    # Converting to cp for FSx/Local
+    cp ~{bam_file} ~{sample_id}.bam
```

Comment:
I think it's dangerous to make a copy of a BAM file as they can be >100GB in some cases. I did notice that this task ran out of disk when I tested, even for a relatively small BAM file. Can we soft-link instead?

Reply:
@epiercehoffman we have added rm at the end and changed cp to mv to facilitate both type of runs. After double checking the bams are not used after this process so it was safe to add rm here.

Reply:
`mv` is fine on GCP, and `rm` at the end is fine but unnecessary as the local copy of the file will be deleted right afterwards, but for a shared file system I want to make sure you realize that MELT should be running in parallel with Manta and Wham and PESRCollection, which all also use the BAM. I imagine that is why you wanted to use `cp` in the first place. Again, happy to use `mv` here for GCP, but wanted to double check that this will accomplish your goals.

Reply:
@epiercehoffman , yes but the good thing is MELT process runs longer than Manta, Whamg etc. so the removal as last step didnt affect our run.

Reply:
@epiercehoffman : I guess now I see/understand what you are talking about. Could we have a solution for this in similar way as we are discussing for ImportGVCF and GEnotypeVCF as above ?

Reply:
@mwalker174 : Similar flag usage here would help both parties.